package.json

```diff
@@ -12,6 +12,7 @@
     "start:production": "node dist/server/index.js",
     "build": "npm run build:css && npm run build:client && npm run build:server",
     "build:css": "node -r ts-node/register server/sass.ts",
+    "build:sass": "echo 'Did you mean build:css?'",
```

Comment:
Could just run the `build:css` script.

Reply:
I did at first bur realized stuff like "npm run build:sass -- --watch" didn't work as expected (the argument wasn't pushed down properly). Not sure if there's a way to pass npm arguments as is.

Reply:
I meant literally to copy the command for `build:css`. Duplication, but it works.